optimum/habana/transformers/models/video_llava/processing_video_llava.py

```diff
@@ -0,0 +1,100 @@
+from typing import List, Optional, Union
+import numpy as np
+
+from transformers.models.video_llava.processing_video_llava import VideoLlavaProcessor
+from transformers.image_processing_utils import BatchFeature
+from transformers.image_utils import ImageInput, get_image_size, to_numpy_array
+from transformers.tokenization_utils_base import PaddingStrategy, PreTokenizedInput, TextInput, TruncationStrategy
+from transformers.utils import TensorType
+from optimum.utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+
+class GaudiVideoLlavaProcessor(VideoLlavaProcessor):
+    attributes = ["image_processor", "tokenizer"]
+    valid_kwargs = ["chat_template", "patch_size", "vision_feature_select_strategy", "image_token", "video_token"]
+    image_processor_class = "VideoLlavaImageProcessor"
+    tokenizer_class = "AutoTokenizer"
+
+    def __init__(
```

Comment:
are you changing the original constructor? It is unclear as You did not mention which version this adaptation is taken from